js/ui/lookup.js

```diff
@@ -623,6 +629,93 @@ var Lookup = DropDownList.inherit({
                 options: {
                     useInkRipple: true
                 }
+            },
+            {
+                device: function() {
+                    return themes.isMaterial();
+                },
+                options: {
+
+                    /**
+                    * @name dxLookupOptions_usePopover
+                    * @publicName usePopover
+                    * @type boolean
+                    * @default false @for Material
+                    */
+                    usePopover: false,
+
+                    /**
+                    * @name dxLookupOptions_closeOnOutsideClick
+                    * @publicName closeOnOutsideClick
+                    * @type boolean|function
+                    * @default true @for Material
+                    * @type_function_return boolean
+                    */
+
+                    closeOnOutsideClick: true,
+
+                    /**
+                     * @name dxLookupOptions_popupWidth
+                     * @publicName popupWidth
+                     * @type number|string|function
+                     * @default undefined @for Material
+                     * @type_function_return number|string
+                     */
+
+                    popupWidth: (function() { return this._$element.outerWidth(); }).bind(this),
+
+                    /**
+                     * @name dxLookupOptions_popupHeight
+                     * @publicName popupHeight
+                     * @type number|string|function
+                     * @default undefined @for Material
+                     * @type_function_return number|string
+                     */
+
+                    popupHeight: (function() { return this._setPopupHeight(MATERIAL_LOOKUP_LIST_ITEMS_COUNT); }).bind(this),
```

Comment:
>_setPopupHeight

It's confusing. Function's name tells that it is a setter but in fact, it's a getter?